index.html

```diff
@@ -46,4 +46,238 @@ <h1 class="mola">  I'm Sayed Ramin<br />Glad to see<br />
         </li>
     </ul>
 </div>
-</section>
\ No newline at end of file
+</section>
+<!--Works Section page 1-->
+<section class="page1">
```

Comment:
- [ ] kindly consider making sure that every opening tag has its closing tag to follow html and css best practices  and that will make your code profesional